extensions/amp-next-page/1.0/service.js

```diff
@@ -804,9 +804,10 @@ export class NextPageService {
    * Add the provided page metadata into the queue of
    * pages to fetch
    * @param {!Array<!./page.PageMeta>} pages
+   * @param {boolean} preventFetching
    * @return {!Promise}
    */
-  queuePages_(pages) {
+  queuePages_(pages, preventFetching = false) {
```

Comment:
Instead of using a boolean parameter, can we rename this method `fetchAndQueuePages` which calls a more concise version of `queuePages_` that we can use above?